glusterfsd/src/glusterfsd-mgmt.c

```diff
@@ -2875,8 +2875,6 @@ glusterfs_mgmt_init(glusterfs_ctx_t *ctx)
     if (!options)
         goto out;
 
-    LOCK_INIT(&ctx->volfile_lock);
```

Comment:
where is it init'ed (and where is it destroyed)?

Reply:
Initialized in `glusterfs_ctx_new()`, should be freed (oops, I forgot about it) in `glusterfs_ctx_destroy()` (the latter is meaningful only for `gfapi` clients).